configure.ac

```diff
@@ -170,6 +182,18 @@ AC_LINK_IFELSE(
 	])
 AC_LANG_POP(C)
 
+dnl ----------
+dnl Essentials
+dnl ----------
+
+AX_PTHREAD([
```

Comment:
The difference is in the command line flag, SEARCH_LIBS results in `-lpthreads` while AX_PTHREAD results in `-pthreads`.  The latter is "pedantically correct"; the difference is that `-pthreads` also tells the compiler that MT is in use and may affect other code generation/behaviour.

(This used to make a difference 10 years ago with shitty compilers; not sure if there are things these days that still break. It may be relevant for thread-local storage qualifiers.)